internal/tools/go.mod

```
@@ -1,10 +1,12 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/internal/tools
 
-go 1.22.1
+go 1.22.5
```

I think this should be 1.22.0?
```suggestion
go 1.22.0
```

---

See: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37700#discussion_r1942449762